app/controllers/invites_controller.rb

```diff
@@ -17,9 +17,13 @@ def show
 
     invite = Invite.find_by(invite_key: params[:id])
     if invite.present? && !invite.expired? && !invite.redeemed?
+      # Hide email only if it mismatches the one provided by Omniauth
+      email = invite.email == session.dig(:authentication, :email) ? invite.email
```

Comment:
I think I would prefer something like:

```
if session[:authentication]
  auth_result = Auth::Result.from_session_data(session[:authentication])
  auth_result.email
end
```
It will achieve almost the same thing, but it uses the `downcase` stuff in Auth::Result, and keeps all our 'session parsing' stuff in one place.

Note that the email provided in this method is not necessarily "verified"... but I think that's ok in this case. We handle the security side of things later.